src/outfit.ts

```diff
@@ -205,6 +214,23 @@ export function meatOutfit(embezzlerUp: boolean, requirement?: Requirement, sea?
       forceEquip.push($item`Kramco Sausage-o-Matic™`);
     }
   }
+
+  if (embezzlerUp) {
+    const currentWeapon = 25 * findLeprechaunMultiplier(meatFamiliar());
+    const value = ((75 - currentWeapon) * (750 + baseMeat)) / 100;
+    const embezzlers = globalOptions.ascending
+      ? 30
+      : Math.min(30, embezzlerCount() || digitizedMonstersRemaining());
+    if (value * embezzlers > mallPrice($item`scratch 'n' sniff UPC sticker`)) {
```

Comment:
`value` here is the value-per-adventure of a 3-sticker sword. Why are we comparing this to the price of a single sticker?

Also, if my suggested code above is used, then the RHS of this equation should be the number of stickers we need to add to our sword (i.e. the additional cost), and not just a flat 3.